### ISSUE TITLE
Switch button for Night mode

### ISSUE THREAD

**Issue by aliasimkazmi** [michaella23/bootcamp-collab#76]
As it is now, the night mode is an oval.
It can be changed into a switch button.

**Comment by aliasimkazmi** [comment]
I'd like to take a look at tis

@michaella23 


I can coordinate with the developer(s) who have worked on the night/dark theme. Info will be appreciated

**Comment by michaella23** [comment]
@aliasimkazmi , yes you can. And I appreciate you reaching out to the other developer. 🙂

**Comment by Akash1965** [comment]
Hi @michaella23 can I work on this issue? Would love to contribute

**Comment by michaella23** [comment]
Hi @aliasimkazmi ! Do you still plan to work on this issue? 🙂

**Comment by aliasimkazmi** [comment]
You can assign it to another developer, if they want to work on this.

**Comment by will-s-205** [comment]
Thanks @aliasimkazmi. 
@michaella23 can I take a look please?
